mbuild/formats/lammpsdata.py

```diff
@@ -301,11 +301,17 @@ def write_lammpsdata(
             mins=mins, maxs=maxs, angles=structure.box[3:6]
         )
     else:
-        # Internally use nm
-        box = Box(
-            lengths=np.array([0.1 * val for val in structure.box[0:3]]),
-            angles=structure.box[3:6],
-        )
```

Comment:
Here, the box length is multiplied by 0.1 to convert from A to nm whether the `unit_style` is `lj` or `real`. Pretty sure this factor should only be applied when the `unit style` is real`.